third_party/move/tools/move-package/src/source_package/std_lib.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::source_package::parsed_manifest::{Dependency, GitInfo};
+use move_command_line_common::env::MOVE_HOME;
+use move_symbol_pool::symbol::Symbol;
+use std::path::PathBuf;
+
+/// Represents a standard library.
+pub enum StdLib {
+    AptosFramework,
+    AptosStdlib,
+    MoveStdlib,
+}
+
+impl StdLib {
+    /// The well-known git URL for the standard library.
+    const STD_GIT_URL: &'static str = "https://github.com/aptos-labs/aptos-core.git";
+
+    /// Returns the dependency for the standard library with the given version.
+    pub fn dependency(&self, version: &StdVersion) -> Dependency {
+        let local =
+            PathBuf::from(MOVE_HOME.clone()).join(format!("{}_{}", self.as_str(), version.rev()));
```

Comment:
Is this the same location which is chosen if someone explicitly uses the github dep in their Move.toml? I'm worried that we are potentially double caching the package here. The local path should be the same to avoid this.

Reply:
done